diana-api/src/main/java/org/apache/diana/api/reader/DateReader.java

```diff
@@ -0,0 +1,32 @@
+package org.apache.diana.api.reader;
+
+
+import org.apache.diana.api.ReaderField;
+
+import java.util.Date;
+
+/**
+ * Class to reads and converts Date type
+ *
+ * @author Daniel Cunha <danielsoro@apache.org>
+ */
+public final class DateReader implements ReaderField {
+
+    @Override
+    public boolean isCompatible(Class clazz) {
+        return Date.class.equals(clazz);
+    }
+
+    @Override
+    public <T> T read(Class<T> clazz, Object value) {
```

Comment:
I think that we will have a different implementation for that. 

Maybe, Date and Calendar should share the same implementation. So, one class for Date and Calendar should be enough.

About Temporal, this could include: LocalDate, LocalDateTime, Year and etc.

What do you think?